NEWS.md

```diff
@@ -1,251 +0,0 @@
-## DataFrames v0.6.11 Release Notes
```

Comment:
Given the lack of upkeep I like your suggestion to just remove it. All of the information in the NEWS.md file can be conveyed in the [release notes](https://github.com/JuliaStats/DataFrames.jl/releases) anyway, right?